src/System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj

```diff
@@ -35,6 +35,7 @@
       <Project>{1e689c1b-690c-4799-bde9-6e7990585894}</Project>
       <Name>System.Runtime</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\System.Threading.Tasks\pkg\System.Threading.Tasks.pkgproj" />
```

Comment:
Nit: You missed to add the ToDo: comment in here. It is important so that we can find them easily in the future so we can remove all at once.